[WWST-6110, WWST-6146] EZEX fingerprint of zigbee-switch.groovy file.

juhaki opened the pull request:
Please add the EZEX fingerprint of the zigbee-switch.groovy file for the EZEX switch ( component 1 )

		// EZEX
		fingerprint profileId: "0104", deviceId: "0100", endpoint: "01", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", manufacturer: "EZEX", model: "E220-KR1N0Z0-HA", deviceJoinName: "EZEX Switch" // 1

juhaki committed:
EZEX fingerprint of zigbee-switch.groovy file.

Please add the EZEX fingerprint of the zigbee-switch.groovy file for the EZEX switch ( component 1 )

		// EZEX
		fingerprint profileId: "0104", deviceId: "0100", endpoint: "01", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", manufacturer: "EZEX", model: "E220-KR1N0Z0-HA", deviceJoinName: "EZEX Switch" // 1

tpmanley commented on devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy:
```diff
@@ -82,6 +82,9 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0019", manufacturer: "Third Reality, Inc", model: "3RSS008Z", deviceJoinName: "RealitySwitch Plus"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0019", manufacturer: "Third Reality, Inc", model: "3RSS007Z", deviceJoinName: "RealitySwitch"
+		
+		// EZEX
```
@juhaki please move this section up so it's in alphabetical order by manufacturer

juhaki replied:
Ok. I will patch to this about alphabetical order move this section.

tpmanley commented on devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy:
```diff
@@ -82,6 +82,9 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0019", manufacturer: "Third Reality, Inc", model: "3RSS007Z", deviceJoinName: "RealitySwitch"
+		
+		// EZEX
+		fingerprint profileId: "0104", deviceId: "0100", endpoint: "01", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", manufacturer: "EZEX", model: "E220-KR1N0Z0-HA", deviceJoinName: "EZEX Switch"
```
You can remove the deviceId and endpoint to more closely match the format used by the other fingerprints in this file

juhaki replied:
After the testing by removed deviceId & endpoint items.
I will remove the deviceId & endpoint.

juhaki committed:
EZEX fingerprint of zigbee-switch.groovy file.

EZEX fingerprint of zigbee-switch.groovy file.
- alphabetical order by manufacturer.
- remove the deviceID & endpoint.
